packages/components/pie-lottie-player/src/index.ts

```diff
@@ -0,0 +1,206 @@
+import { LitElement, html, unsafeCSS } from 'lit';
+import { property, query } from 'lit/decorators.js';
+import { LottiePlayer, AnimationItem } from 'lottie-web';
+
+import { defineCustomElement } from '@justeattakeaway/pie-webc-core';
+import { LottiePlayerProps, defaultProps, AnimationDirection } from './defs';
+
+// Valid values available to consumers
+export * from './defs';
+
+const componentSelector = 'pie-lottie-player';
+
+/**
+ * @tagname pie-lottie-player
+ */
+export class PieLottiePlayer extends LitElement implements LottiePlayerProps {
+    @query('div')
+    private _hostElement!: HTMLDivElement;
+
+    private _lottie?:LottiePlayer;
+    private _animationInstance?: AnimationItem;
+
+    private _animationSrc = '';
+    private _animationData?:object;
+    private _autoPlayDisabled = defaultProps.autoPlayDisabled;
+    private _loopDisabled = defaultProps.loopDisabled;
+    private _speed = defaultProps.speed;
+    private _direction = defaultProps.direction;
+
+    protected async firstUpdated (): Promise<void> {
+        // Check if the code is running in a browser environment
+        if (this._canUseDOM()) {
+            const { default: lottieModule } = await import('lottie-web/build/player/lottie_light_canvas.min.js'); // Dynamically import the 'lottie-web' library to avoid SSR issues
+            this._lottie = lottieModule;
+            this._loadAnimation();
+        }
+    }
+
+    disconnectedCallback () {
+        super.disconnectedCallback();
+        this._destroyAnimation();
+    }
+
+    private _canUseDOM () {
+        return typeof window !== 'undefined' && 'document' in window && 'createElement' in window.document;
+    }
```

Comment:
would the Lit `isServer` directive be useful here instead? [Reference](https://lit.dev/docs/api/misc/#isServer)